process-migration-service/src/test/resources/project-test.yml

```diff
@@ -0,0 +1,30 @@
+thorntail:
```

Comment:
Just out of curiosity. Do we have a test stage or -Stest configuration anywhere? I cannot see it. What is the difference between stages and -S switch?